### ISSUE TITLE
Can't build enunciate - setting the executable properties to true doesn't work

### ISSUE THREAD

**Issue by chibi03** [stoicflame/enunciate#185]
Hi, 
I'm trying to build enunciate myself and set the gmcs.executable property to true as indicated by the error message but I got the following error as a result:
(I added `<gmcs.executable>true</gmcs.executable>` to the top-level pom.xml)
```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 55.393 s
[INFO] Finished at: 2015-10-16T10:47:50+02:00
[INFO] Final Memory: 70M/740M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.3:run (enunciate) on project enunciate-csharp-xml-client: An Ant BuildException has occured: java.lang.RuntimeException: com.webcohesion.enunciate.EnunciateException: java.io.IOException: Cannot run program "null" (in directory "C:\workspace\enuncicate\enunciate-master\csharp-xml-client\target\enunciate\csharp-xml-client\compile"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden: OnError while emitting onNext value: com.webcohesion.enunciate.EnunciateContext.class -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.3:run (enunciate) on project enunciate-csharp-xml-client: An Ant BuildException has occured: java.lang.RuntimeException: com.webcohesion.enunciate.EnunciateException: java.io.IOException: Cannot run program "null" (in directory "C:\workspace\enuncicate\enunciate-master\csharp-xml-client\target\enunciate\csharp-xml-client\compile"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:216)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
        at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
        at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
        at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
        at org.apache.maven.cli.MavenCli.execute(MavenCli.java:862)
        at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:286)
        at org.apache.maven.cli.MavenCli.main(MavenCli.java:197)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
        at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
        at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoExecutionException: An Ant BuildException has occured: java.lang.RuntimeException: com.webcohesion.enunciate.EnunciateException: java.io.IOException: Cannot run program "null" (in directory "C:\workspace\enuncicate\enunciate-master\csharp-xml-client\target\enunciate\csharp-xml-client\compile"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at org.apache.maven.plugin.antrun.AbstractAntMojo.executeTasks(AbstractAntMojo.java:131)
        at org.apache.maven.plugin.antrun.AntRunMojo.execute(AntRunMojo.java:98)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
        ... 20 more
Caused by: java.lang.RuntimeException: com.webcohesion.enunciate.EnunciateException: java.io.IOException: Cannot run program "null" (in directory "C:\workspace\enuncicate\enunciate-master\csharp-xml-client\target\enunciate\csharp-xml-client\compile"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:116)
        at org.apache.tools.ant.Task.perform(Task.java:348)
        at org.apache.tools.ant.Target.execute(Target.java:357)
        at org.apache.maven.plugin.antrun.AbstractAntMojo.executeTasks(AbstractAntMojo.java:118)
        ... 23 more
Caused by: java.lang.RuntimeException: com.webcohesion.enunciate.EnunciateException: java.io.IOException: Cannot run program "null" (in directory "C:\workspace\enuncicate\enunciate-master\csharp-xml-client\target\enunciate\csharp-xml-client\compile"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at com.sun.tools.javac.main.Main.compile(Main.java:469)
        at com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:132)
        at com.webcohesion.enunciate.Enunciate.run(Enunciate.java:651)
        at com.webcohesion.enunciate.EnunciateTask.execute(EnunciateTask.java:155)
        at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:288)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:606)
        at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
        ... 26 more
Caused by: com.webcohesion.enunciate.EnunciateException: java.io.IOException: Cannot run program "null" (in directory "C:\workspace\enuncicate\enunciate-master\csharp-xml-client\target\enunciate\csharp-xml-client\compile"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at com.webcohesion.enunciate.modules.csharp_client.CSharpXMLClientModule.compileSources(CSharpXMLClientModule.java:452)
        at com.webcohesion.enunciate.modules.csharp_client.CSharpXMLClientModule.call(CSharpXMLClientModule.java:126)
        at com.webcohesion.enunciate.io.InvokeEnunciateModule.onNext(InvokeEnunciateModule.java:31)
        at com.webcohesion.enunciate.io.InvokeEnunciateModule.onNext(InvokeEnunciateModule.java:10)
        at rx.internal.operators.OperatorDoOnEach$1.onNext(OperatorDoOnEach.java:79)
        at rx.internal.operators.OperatorSingle$ParentSubscriber.onCompleted(OperatorSingle.java:109)
        at rx.internal.operators.NotificationLite.accept(NotificationLite.java:140)
        at rx.internal.operators.TakeLastQueueProducer.emit(TakeLastQueueProducer.java:100)
        at rx.internal.operators.TakeLastQueueProducer.startEmitting(TakeLastQueueProducer.java:45)
        at rx.internal.operators.OperatorTakeLast$1.onCompleted(OperatorTakeLast.java:59)
        at rx.internal.operators.OperatorMerge$MergeSubscriber.drainAndComplete(OperatorMerge.java:521)
        at rx.internal.operators.OperatorMerge$MergeSubscriber.completeInner(OperatorMerge.java:486)
        at rx.internal.operators.OperatorMerge$InnerSubscriber.emit(OperatorMerge.java:658)
        at rx.internal.operators.OperatorMerge$InnerSubscriber.onCompleted(OperatorMerge.java:600)
        at rx.subjects.SubjectSubscriptionManager$SubjectObserver.onCompleted(SubjectSubscriptionManager.java:232)
        at rx.subjects.ReplaySubject.onCompleted(ReplaySubject.java:404)
        at rx.Observable$35.onCompleted(Observable.java:7458)
        at rx.observers.SafeSubscriber.onCompleted(SafeSubscriber.java:81)
        at rx.internal.operators.OperatorDoOnEach$1.onCompleted(OperatorDoOnEach.java:53)
        at rx.internal.operators.OperatorSingle$ParentSubscriber.onCompleted(OperatorSingle.java:110)
        at rx.internal.operators.NotificationLite.accept(NotificationLite.java:140)
        at rx.internal.operators.TakeLastQueueProducer.emit(TakeLastQueueProducer.java:100)
        at rx.internal.operators.TakeLastQueueProducer.startEmitting(TakeLastQueueProducer.java:45)
        at rx.internal.operators.OperatorTakeLast$1.onCompleted(OperatorTakeLast.java:59)
        at rx.internal.operators.OperatorMerge$MergeSubscriber.drainAndComplete(OperatorMerge.java:521)
        at rx.internal.operators.OperatorMerge$MergeSubscriber.completeInner(OperatorMerge.java:486)
        at rx.internal.operators.OperatorMerge$InnerSubscriber.emit(OperatorMerge.java:658)
        at rx.internal.operators.OperatorMerge$InnerSubscriber.onCompleted(OperatorMerge.java:600)
        at rx.internal.util.ScalarSynchronousObservable$ScalarSynchronousAction.call(ScalarSynchronousObservable.java:117)
        at rx.schedulers.ExecutorScheduler$ExecutorAction.run(ExecutorScheduler.java:173)
        at rx.schedulers.ExecutorScheduler$ExecutorSchedulerWorker.run(ExecutorScheduler.java:99)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Cannot run program "null" (in directory "C:\workspace\enuncicate\enunciate-master\csharp-xml-client\target\enunciate\csharp-xml-client\compile"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1047)
        at com.webcohesion.enunciate.modules.csharp_client.CSharpXMLClientModule.compileSources(CSharpXMLClientModule.java:432)
        ... 33 more
Caused by: java.io.IOException: CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at java.lang.ProcessImpl.create(Native Method)
        at java.lang.ProcessImpl.<init>(ProcessImpl.java:385)
        at java.lang.ProcessImpl.start(ProcessImpl.java:136)
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1028)
        ... 34 more
Caused by: rx.exceptions.OnErrorThrowable$OnNextValue: OnError while emitting onNext value: com.webcohesion.enunciate.EnunciateContext.class
        at rx.exceptions.OnErrorThrowable.addValueAsLastCause(OnErrorThrowable.java:101)
        at rx.internal.operators.OperatorDoOnEach$1.onNext(OperatorDoOnEach.java:81)
        ... 29 more
[ERROR]
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :enunciate-csharp-xml-client
```

Thanks for your help!

**Comment by stoicflame** [comment]
Okay. Thanks for the report.

**Comment by stoicflame** [comment]
Fixed at 59698e1. Please validate.

**Comment by stoicflame** [comment]
[Enunciate 2.1 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.1.0)
